plugins/uv/uv.plugin.zsh

```diff
@@ -0,0 +1,38 @@
+# Return immediately if uv is not found
+if ((!$ + commands[uv])); then
```

Comment:
Make sure syntax is correct here
```suggestion
if (( ! ${+commands[uv]} )); then
```

Reply:
I know very little about ZSH, really. I'm getting this error:

<img width="498" alt="image" src="https://github.com/user-attachments/assets/ff8c5050-7ef9-430d-aa35-b84698a36fcf">

But the script works when manually copied into `custom/plugins/uv`. I'd guess it's actually correct, but let me know if something isn't working as expected!